src/cursor/cur_std.c

```diff
@@ -281,10 +281,15 @@ __wt_cursor_get_key(WT_CURSOR *cursor, ...)
 void
 __wt_cursor_set_key(WT_CURSOR *cursor, ...)
 {
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
     va_list ap;
 
+    session = CUR2S(cursor);
```

Comment:
If we only use session once, we can directly use CUR2S no need to define a variable.

Reply:
Fixed.